client/components/search/index.jsx

```diff
@@ -349,6 +346,10 @@ class Search extends Component {
 					<input
 						type="search"
 						id={ 'search-component-' + this.instanceId }
+						autoFocus={ this.props.autoFocus } // eslint-disable-line jsx-a11y/no-autofocus
```

Comment:
is this against recommendations?

Reply:
This triggers a warning, but it's the suggested solution on https://github.com/Automattic/wp-calypso/issues/21658.

The problem is that `autoFocus` could prevent screen readers to read the content before the component, which in this case we are using `aria-describedby` to reference, so it's announced correctly by screen readers.

Reply:
Let's add another comment explaining this. Folks might try to remove this otherwise.